src/main/java/io/supertokens/webserver/api/totp/VerifyTotpAPI.java

```diff
@@ -76,13 +70,10 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
                 tenantIdentifierWithStorage = getTenantIdentifierWithStorageFromRequest(req);
             }
 
-            Totp.verifyCode(tenantIdentifierWithStorage, main, userId, totp, allowUnverifiedDevices);
+            Totp.verifyCode(tenantIdentifierWithStorage, main, userId, totp, false);
```

Comment:
if we are never calling verifyCode with true, then we should just remove the `allowUnverifiedDevices` boolean from the input of the function.

Reply:
Done